example/src/config/i18n.js

```diff
@@ -0,0 +1,61 @@
+import i18n from 'i18next';
+import { initReactI18next } from 'react-i18next';
+import en from '../langs/en.json';
+import fr from '../langs/fr.json';
+
+i18n.use(initReactI18next).init({
+  resources: {
+    en,
+    fr,
+  },
+  lng: 'en',
+  fallbackLng: 'en',
+  // debug only when not in production
+  debug: process.env.NODE_ENV !== 'production',
+  ns: ['translations'],
+  defaultNS: 'translations',
+  keySeparator: false,
+  interpolation: {
+    escapeValue: false,
+    formatSeparator: ',',
+  },
+  react: {
+    wait: true,
+  },
+});
+
+const langs = {
+  // bg: "български",
+  // ca: "Català",
+  // cs: "čeština",
+  // de: "Deutsch",
+  // el: "Ελληνικά",
+  en: 'English',
+  // es: "Español",
+  // et: "Eesti",
+  // fi: "Suomi",
+  fr: 'Français',
+  // hu: "Magyar",
+  // it: "Italiano",
+  // ja: '日本語',
+  // ka: "ქართული",
+  // lt: "lietuvių kalba",
+  // lv: "Latviešu",
+  // nl: "Nederlands",
+  // pt: "Português",
+  // ro: "Română",
+  // ru: "Русский",
+  // sk: "Slovenský",
+  // sl: "Slovenščina",
+  // sr: "српски језик",
+  // sw: 'Kiswahili',
+  // tr: "Türkçe",
+  // uk: "Українська",
+  // vi: "Tiếng Việt",
+  // zh: "简体中文",
+  // zh_tw: "繁體中文",
+};
+
+export { langs };
+
+export default i18n;
```

Comment:
I'm surprised you need translations, do you need this file?

Reply:
No, it was just to add it for the future, but i will remove it and we can add it when we need